packages/slice-machine/server/src/api/simulator.ts

```diff
@@ -38,20 +65,8 @@ export default async function handler(
   const { dependencies, devDependencies } = packageJson.content;
   // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
   const deps: Record<string, string> = { ...dependencies, ...devDependencies };
-
-  const requiredDeps =
-    req.env.framework === Frameworks.next
-      ? [
-          Utils.CONSTS.SLICE_SIMULATOR_REACT,
-          Utils.CONSTS.NEXT_SLICEZONE,
-          Utils.CONSTS.PRISMIC_REACT_PACKAGE_NAME,
-        ]
-      : [
-          Utils.CONSTS.SLICE_SIMULATOR_VUE,
-          Utils.CONSTS.NUXT_SM,
-          Utils.CONSTS.VUE_SLICEZONE,
-          Utils.CONSTS.NUXT_PRISMIC,
-        ];
+  // TODO: fix this for legacy frameworks, and svelte?
```

Comment:
don't get it, svelte and other frameworks doesn't support the simulator why we should handle them at this point ?

Reply:
good point, what about for legacy nuxt?

Reply:
It's in another task to support it :)